bitnami/mongodb/README.md

```diff
@@ -110,6 +110,7 @@ The following table lists the configurable parameters of the MongoDB chart and t
 | `priorityClassName`                                | Pod priority class name                                                                                                                                   | ``                                                       |
 | `extraEnvVars`                                     | Array containing extra env vars to be added to all pods in the cluster (evaluated as a template)                                                          | `nil`                                                    |
 | `nodeSelector`                                     | Node labels for pod assignment                                                                                                                            | `{}`                                                     |
+| `nodeSelectorArbiter`                              | Node labels for arbiter pod assignment                                                                                                                    | `{}`                                                     |
```

Comment:
please say that `nodeSelectorArbiter` will override `nodeSelector`

Reply:
done